src/components/content/section.tsx

```diff
@@ -0,0 +1,24 @@
+import { DivProps } from '@powerhousedao/design-system';
+import React from 'react';
+
+export interface ContentSection extends DivProps {
+    title?: string;
+    children?: React.ReactNode;
```

Comment:
`children` is already part of DivProps, not important though

Reply:
right! fix here: [9f6a4ac](https://github.com/powerhouse-inc/document-model-electron/pull/46/commits/9f6a4ac45318bb757e7c7c60df463e67b066771b)